install

```diff
@@ -12,8 +12,8 @@ CA_KEY_PATH=$KEY_DIR/Lamassu_OP_Root_CA.key
 CA_PATH=$CERT_DIR/Lamassu_OP_Root_CA.pem
 SERVER_KEY_PATH=$KEY_DIR/Lamassu_OP.key
 SERVER_CERT_PATH=$CERT_DIR/Lamassu_OP.pem
-SEEDS_DIR=$HOME/seeds
-SEED_FILE=$SEEDS_DIR/seed.txt
+MNEMONIC_DIR=$HOME/mnemonics
```

Comment:
Should be ``/etc/lamassu/mnemonics``

Reply:
Even thought it was previously saving the seeds on $HOME/seeds?

Reply:
Hmm, I see. Yeah, I think /etc/lamassu is a better place for it and more consistent with where we're putting other stuff. Sorry about the confusion.